packages/neuron-wallet/src/utils/ckb-rpc.ts

```diff
@@ -286,7 +330,9 @@ export class LightRPC extends Base {
             if (res.id !== payload[i].id) {
               return new IdNotMatchedInBatchException(i, payload[i].id, res.id)
             }
-            return ({ ...rpcProperties, ...lightRPCProperties })[proxied[i][0]].resultFormatters?.(res.result) ?? res.result
+            return (
+              { ...rpcProperties, ...lightRPCProperties }[proxied[i][0]].resultFormatters?.(res.result) ?? res.result
+            )
```

Comment:
I'm not sure if this formatting(and some elsewhere) is introduced by local lint rule

Reply:
I unconsciously prettier this file by a shortcut, instead of the complaint from eslint

Because of this comment, I realized that the prettier seems doesn't work with eslint since the `eslint-plugin-prettier` was not installed

Reply:
I found that we have not yet checked for prettier in lint, and after this PR merge, I think we should open another PR by adding `plugin:prettier/recommended` to the `.eslintrc.js` configuration

Reply:
Will be handled by https://github.com/nervosnetwork/neuron/pull/2692